### ISSUE TITLE
WCAG: Esc works for a vertical volume bar 

### ISSUE THREAD

**Issue by gjanblaszczyk** [videojs/video.js#6004]
## Description
According to WCAG 2.1 and 1.4.13 Content on Hover or Focus.
We should be able to dismiss a **vertical** volume bar (vidoejs by default uses horizontal volume slider) and probably captions/subtitles menus with Esc key when they are shown after hovering the mouse.

## Steps to reproduce
Explain in detail the exact steps necessary to reproduce the issue.
1. Use the mouse to hover the volume button
2. You should be able to see the vertical volume slider
3. Press the Esc button and you will see that the volume bar slider is still visible

### Expected
Pressing the Esc key will hide the vertical volume bar. 

### Actual
Pressing the Esc key doesn't hide the vertical volume bar.

## Additional Information
Please include any additional information necessary here. Including the following:
#### videojs
7.5.4
#### browsers
All
#### OSes
All
### plugins
no plugins

**Comment by gkatsev** [comment]
Interesting, this kind of makes sense. This potentially should apply to any of our menus as well?

**Comment by gjanblaszczyk** [comment]
Yes, exactly. The changes should also apply to the menus.

**Comment by gjanblaszczyk** [comment]
@gkatsev I created PR for this issue but I am not 100% sure if that is a good approach, will be great if you can give me some feedback about it.

**Comment by gkatsev** [comment]
Thanks, I'll take a look but may not happen until next week.

**Comment by gkatsev** [comment]
Unfortunately, it's likely that I won't be able to look at this PR this week either.

**Comment by gjanblaszczyk** [comment]
Thanks for the info and that's fine. It is not urgent anyway.
